my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
```

Comment:
Could you generalize this to get the filename from the command-line? You could do:

```
import sys
filename = sys.argv[1]
data = fits.getdata(filename)
```